engine/src/main/java/org/camunda/bpm/engine/impl/persistence/entity/HistoricProcessInstanceManager.java

```diff
@@ -271,6 +271,7 @@ public void addRemovalTimeById(String processInstanceId, Date removalTime) {
 
   public Map<Class<? extends DbEntity>, DbOperation> deleteHistoricProcessInstancesByRemovalTime(Date removalTime, int minuteFrom, int minuteTo, int batchSize) {
     CommandContext commandContext = Context.getCommandContext();
+    commandContext = null;
```

Comment:
❓ @psavidis. is this `null` intentional? Judging by the failing CI, I would I assume that it's overlooked. Could you please double check before I give my final review?

Reply:
It was committed by mistake due to context switching. After the CI passes, the review can continue. Sorry for the inconvenience.